Design/SANS/BeamCentreFinderDesign.md

```diff
@@ -0,0 +1,119 @@
+# Design Document for Beam Centre Finder
+
+### Current issue
+Currently the Beam Centre Finder(BCF) is heavily tied up with the ISIS instruments and the ReductionSingleton().
+ANSTO are looking currently for similar functionality but since their reduction workflow is not based on the
+ReductionSingleton() and they don't make use of the instrument class they cannot use the BCF directly. 
+This document lines out the required changes to make the BCF useable in a more general context which could 
+also be beneficial when moving away from the current Reduction-Step approach.
+
```

Comment:
Can you add a *Benefits* summary to justify the work. Include lower maintenace overhead for both facilities etc.